Tests/DeepLearningTests/LayerTests.swift

```diff
@@ -134,15 +134,12 @@ final class LayerTests: XCTestCase {
       let size = 6
       let layer = UpSampling3D<Float>(size: size)
       let input = Tensor<Float>(shape: [1, 4, 3, 2, 1], scalars: (0..<24).map(Float.init))
-      // TODO(TF-525): Fix `UpSampling3D.call`.
-      // Broadcasting does not support tensors with high rank:
-      // Broadcast between [1,4,1,3,1,2,1,1] and [1,1,6,1,6,1,6,1] is not supported yet.
-      /*
+      // TODO: Fix bug in `UpSampling3D.repeatingElements`.
+      // Result has shape `[1, 96, 54, 24, 1]` instead of `[1, 24, 18, 12, 1]`.
```

Comment:
Todo: fix the bug in `UpSampling3D.repeatingElements`. I used [`repeat_elements`](https://github.com/tensorflow/tensorflow/blob/0c3ebf695028e8a233cd82b6ba7066a0a131d9f8/tensorflow/python/keras/backend.py#L2634) from Keras as my reference, but the shapes don't work out.

@Shashi456: would you like to continue looking into this?

Reply:
@dan-zheng It's exact height, length and width more times.

Reply:
@Shashi456 Thanks for the catch! The shape mismatch is fixed in https://github.com/tensorflow/swift-apis/pull/131/commits/c6e817c2c3b3d47d7535492fbc957ee4d21c4128. Upsampling tests should be updated to also check correctness of output values, not just output shapes.

Reply:
@dan-zheng Actually that way there a lot more tests to be written for many layers, checking different kinds of functionalities :P. I'll keep that in mind though. Great job, being so quick to fix this.

Reply:
More tests would be awesome!

Reply:
I'll make a comprehensive list in #77, might help newcomers.